src/main/java/org/folio/services/kafka/topic/KafkaTopic.java

```diff
@@ -13,6 +14,10 @@ public static KafkaTopic item(String tenantId, String environmentName) {
     return new KafkaTopic(qualifyName("inventory.item", environmentName, tenantId));
   }
 
+  public static KafkaTopic forName(String name, String tenantId, String environmentName) {
+    return new KafkaTopic(qualifyName(name, environmentName, tenantId));
+  }
```

Comment:
Why is a general method needed for creating any topic name needed?

If it is, could the existing methods use this method as well?

Reply:
It's needed because topic name comes from the request and it could be arbitrary name (see iteration-job-params.json)

> could the existing methods use this method as well?
ok, i'll do that